doc/minimumrequirements.md

```diff
@@ -0,0 +1,22 @@
+# Minimum requirements
+
+Most modern computers will be able to run Hagall. We have tested on desktops, laptops, servers and Raspberry Pis.
+
+- An x86 or ARMv6+ processor
```

Comment:
Should be `x86, x86_64 or ARMv6+`.

Reply:
updated to `An x86, x86_64 or ARMv6+ processor`

Reply:
x86 kinda implies both 32-bit versions and 64-bit versions are supported. ARMv6+ implies ARMv6 (32-bit) and ARMv7/v8 (64-bit) are supported, so not sure we need to add on to the list.